### ISSUE TITLE
AutoFlattenSeq if top level grammar is wrapped in <>

### ISSUE THREAD

**Issue by ajimu** [lbradstreet/instaparse-cljs#16]
I'm not sure I'm wording this right since I'm out of my depth, but the following grammar from the README results in `#object[instaparse.auto-flatten-seq.AutoFlattenSeq]` for me:

``` clojure
(def p (insta/parser "

    <paren-wrapped> = <'('> seq-of-A-or-B <')'>
    <seq-of-A-or-B> = ('a' | 'b')*

"))
```

I'm trying to print the result like this: `(println (p "(aaaa)"))`

**Comment by aengelberg** [comment]
Interesting. It's not surprising that AutoFlattenSeq is being returned (that's Instaparse's own version of seq for efficiency) but looks like the ClojureScript version of that class isn't being printed properly.

**Comment by aengelberg** [comment]
As a workaround before this is fixed, you can instead call `(println (seq (p "(aaaa)")))` to turn it into something printable. That will behave weirdly if the parse result is a failure, however.

**Comment by lbradstreet** [comment]
Thanks for the report, and the fix from @aengelberg. This has been fixed in 1.4.1.2.
